### DISCUSSION TITLE
Programowanie

### DISCUSSION THREAD

**Discussion by dejwodejo** [math-insight/math-insight#3] [Announcements]
Hej, kilka pytań i uzupełnień.
Tu [link](https://roadmap.sh/python) znajdziecie stronkę o której mówiłem, gdzie w uporządkowany sposób możecie zobaczyć jak nauczyć się pythona, ale oczywiście jest to tylko propozycja - najważniejsze samemu grzebać i być ciekawym. Jeśli chcecie, to dajcie znać, to dodam jakąś listę zadań, które można rozwiązać, żeby uczyć się na łatwiejszych i trudniejszych przykładach.
Odnośnie wykładu z Gita, chciałem zaproponować termin 12.06 o 15:00 w sali 205, dajcie znać czy pasuje taki termin, to dam znać dr Niedzieli i prof. Bojarskiemu czy im też taki termin pasuje. Prosiłbym tylko o zainstalowanie gita przed wykładem.

**Comment by dejwodejo** [comment]
Hej ciąg dalszy. Dodałem repozytorium [netflix-dataset](https://github.com/Insight-Applied-Math-UZ/netflix-dataset). Dane możecie ściągnąć tu https://www.kaggle.com/datasets/ashishgup/netflix-rotten-tomatoes-metacritic-imdb. Teraz jaki jest zamysł projektu. Każdy będzie musiał zrobić część programu w pythonie. 
Zadaniem pierwszej osoby, będzie sclonowanie repozytorium do siebie na komputer, utworzenie brancha o odpowiedniej nazwie i przygotowanie środowiska do pythona i utworzenie pliku z skryptem, który będzie wczytywał dane z podanego wcześniej linku

**Comment by bodzaruz** [comment]
Hej, 
Ad. 1. Problemy są niezmienne, także na początek proponuję wziąć, wejściówki i zadania z lista i je rozwiązywać - to jeśli chodzi o Python w analizach. Na pewno przydadzą Wam się biblioteki takie jak: pandas, matplotlib oraz numpy.
Ad. 2. Jeżeli w jakikolwiek krępuje Was moja obecność podczas spotkać tj. ostatnie z Git, to proszę śmiało dać znać. Macie prawo odpocząć ode mnie, a ja się nie pogniewam. Możecie dać znać @BranTheBroken, On mi przekaże.
Ad. 3. Dobry pomysł. @BranTheBroken kto miałby być tą pierwszą osobą? Kto pierwszy ten lepszy?

**Reply by dejwodejo** [reply]
3. Zaczął @marcin2203, jego branch jest widoczny i czeka na 1 małą poprawkę. Wiem, że @MaksLew też jest zainteresowany robieniem tego, jednak po dodaniu brancha Marcina, każdy będzie mógł skolonować repozytorium i dodawać swoje funkcjonalności niezależnie.

**Comment by dejwodejo** [comment]
Siema siema. Dodałem [repozytorium z zadaniami](https://github.com/Insight-Applied-Math-UZ/coding-basics/tree/main). repozytorium należy sforkować do siebie na konto. w razie pytań piszcie do mnie
